src/plugins/vis_types/xy/public/config/get_aspects.ts

```diff
@@ -61,13 +62,13 @@ function getAspectsFromDimension(
   if (Array.isArray(dimensions)) {
     return compact(
       dimensions.map((d) => {
-        const column = d && columns[d.accessor];
+        const column = d && getColumnByAccessor(columns, d.accessor);
```

Comment:
This function is used a lot both in xy and vislib plugin. Can we right some tests (unit) for them?

Reply:
Nick asked for exposing those functions from elastic-charts, so, I didn't want to do that )
https://github.com/elastic/elastic-charts/issues/1379
But I'll write, np.

Reply:
Added tests for most important utils at `accessors.ts`.